doc/release/1.7.0-notes.rst

```diff
@@ -24,53 +24,295 @@ For running on PyPy, PyPy3 6.0+ is required.
+to allow one-sided hypothesis testing.
+
+Sample statistics
+-----------------
+
+The new function `scipy.stats.differential_entropy` estimates the differential
+entropy of a continuous distribution from a sample.
+
+The ``boxcox`` and ``boxcox_normmax`` now allow the user to control the
+optimizer used to minimize the negative log-likelihood function.
+
+A new function `scipy.stats.contingency.relative_risk` calculates the
+relative risk, or risk ratio, of a 2x2 contingency table. The object
+returned has a method to compute the confidence interval of the relative risk.
+
+Performance improvements in the ``skew`` and ``kurtosis`` functions achieved
+by removal of repeated/redundant calculations.
+
+Substantial performance improvements in `scipy.stats.mstats.hdquantiles_sd`.
+
+The new function `scipy.stats.contingency.association` computes several
+measures of association for a contingency table: Pearsons contingency
+coefficient, Cramer's V, and Tschuprow's T.
+
+The parameter ``nan_policy`` was added to `scipy.stats.zmap` to provide options
+for handling the occurrence of ``nan`` in the input data.
+
+The parameter ``ddof`` was added to `scipy.stats.variation` and
+`scipy.stats.mstats.variation`.
+
+The parameter ``weights`` was added to `scipy.stats.gmean`.
+
+Statistical Distributions
+-------------------------
+
+We now vendor and leverage the Boost C++ library to address a number of
+previously reported issues in ``stats``. Notably, ``beta``, ``binom``,
+``nbinom`` now have Boost backends, and it is straightforward to leverage
+the backend for additional functions.
+
+The skew Cauchy probability distribution has been implemented as
+`scipy.stats.skewcauchy`.
+
+The Zipfian probability distribution has been implemented as
+`scipy.stats.zipfian`.
+
+The new distributions ``nchypergeom_fisher`` and ``nchypergeom_wallenius``
+implement the Fisher and Wallenius versions of the noncentral hypergeometric
+distribution, respectively.
+
+The generalized hyperbolic distribution was added in
+`scipy.stats.genhyperbolic`.
+
+The studentized range distribution was added in `scipy.stats.studentized_range`.
+
+`scipy.stats.argus` now has improved handling for small parameter values.
+
+Better argument handling/preparation has resulted in performance improvements
+for many distributions.
+
+The ``cosine`` distribution has added ufuncs for ``ppf``, ``cdf``, ``sf``, and
+``isf`` methods including numerical precision improvements at the edges of the
+support of the distribution.
+
+An option to fit the distribution to data by the method of moments has been
+added to the ``fit`` method of the univariate continuous distributions.
+
+Other
+-----
+`scipy.stats.bootstrap` has been added to allow estimation of the confidence
+interval and standard error of a statistic.
+
+The new function `scipy.stats.contingency.crosstab` computes a contingency
+table (i.e. a table of counts of unique entries) for the given data.
+
+New `scipy.stats.qmc` module
```

Comment:
```suggestion
table (i.e. a table of counts of unique entries) for the given data.

`scipy.stats.NumericalInverseHermite` enables fast random variate sampling
and percentile point function evaluation of an arbitrary univariate statistical
distribution.

New `scipy.stats.qmc` module
```

Reply:
Indeed, If you have some time to do it, that would be great! On my side sorry I am getting short today.

Reply:
Merged